_projects/SP19-Turtle_Racing_Simulator_2000.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: Turtle Racing Simulator 2000
+image: https://i.imgur.com/lESn3hS.jpg
```

Comment:
We'd prefer this picture in `/img/projects/SP19-Turtle_Racing_Simulator.jpg`, if possible.

Reply:
Did that just now, did I link it correctly?

Reply:
yep, thanks!